lib/inc/drogon/HttpTypes.h

```diff
@@ -95,7 +95,8 @@ enum ContentType
     CT_IMAGE_XICON,
     CT_IMAGE_ICNS,
     CT_IMAGE_BMP,
-    CT_MULTIPART_FORM_DATA
+    CT_MULTIPART_FORM_DATA,
+    CT_APPLICATION_WASM
```

Comment:
Thanks a lot.
But It is best to place CT_APPLICATION_WASM in front of CT_APPLICATION_OCTET_STREAM in the enumeration definition to enable compression.
see here:
https://github.com/an-tao/drogon/blob/986ef8dda7c17f848937110c846dabcb2da0f2eb/lib/src/HttpServer.cc#L252-L260

Reply:
Ohh, you're right, I didn't know that the order had something to do with the compression. I force updated the commit to allow compression. I did some testing, and my sample .wasm file with a size of 1,380,991 Bytes was gzipped down to 567,500 Bytes (41%), so it's worth it.

Reply:
And you also could make a static .wasm.gz file for saving bandwidth without increasing CPU load. see #141